docs/CHANGELOG.asciidoc

```diff
@@ -93,6 +94,279 @@ you make the necessary updates after you upgrade to 8.15.0.
+* Changes the behavior of the "Endpoints and API keys" button in the header to open the Connection details flyout ({kibana-pull}183236[#183236]).
+* Adds a quick way to create an API keys with a 90-days expiration to the Connection details flyout, and clarifies the Elasticsearch endpoint and Cloud ID information ({kibana-pull}180912[#180912]).
+* Adds a feedback button to the header in Serverless projects({kibana-pull}180942[#180942]).
+
+For more information about the features introduced in 8.15.0, refer to <<whats-new,What's new in 8.15>>.
+
+[[enhancements-and-bug-fixes-v8.15.0]]
+=== Enhancements and bug fixes
+
+For detailed information about the 8.15.0 release, review the enhancements and bug fixes.
+      
+
+[float]
+[[enhancement-v8.15.0]]
+=== Enhancements
+Alerting::
+* Adds support of additional fields for ServiceNow ITSM and SecOps ({kibana-pull}184023[#184023]).
+* Adds support for the additional info field in the ServiceNow ITOM connector ({kibana-pull}183380[#183380]).
+Cases::
+* The Cases webhook connector now supports SSL certificate authentication ({kibana-pull}185925[#185925]).
+Dashboards::
+* Adds a "Creator" column to the Dashboards list ({kibana-pull}182256[#182256]).
+* Adds the ability to filter dashboards by creator for dashboards created on or after version 8.14 ({kibana-pull}180147[#180147]).
+* When switching back from maximized to minimized view on a panel, you return to your original position in the dashboard ({kibana-pull}184696[#184696]).
+* Improves the dashboard background color to match the current color mode wen margins are turned off ({kibana-pull}181450[#181450]).
+* Simplifies the workflow for creating a copy of the dashboard currently open in both view and edit modes ({kibana-pull}180938[#180938]).
+Data Discovery::
+* CSV reports now include custom field labels when they exist ({kibana-pull}181565[#181565]).
+Discover::
+* Adds the ability to creates ES|QL WHERE filters for ordinal charts ({kibana-pull}184420[#184420]).
+* Adds a button to expand the time range on demand when no results are found for a search ({kibana-pull}181723[#181723]).
+* Changes the Discover document viewer flyout to push the rest of the UI instead of hiding it with an overlay ({kibana-pull}166406[#166406]).
+Elastic Security::
+* For the Elastic Security 8.15.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+ES|QL::
+* Improves metadata autocomplete suggestions with comma and pipe ({kibana-pull}188338[#188338]).
+* Automatically encapsulate index names with special characters with quotes ({kibana-pull}187899[#187899]).
+* Adds support for integrations ({kibana-pull}184716[#184716]).
+* Adds the ability to comment out the current line with the keyboard using `CMD + /` ({kibana-pull}184637[#184637]).
+* Adds the ability to filter an ES|QL chart by brushing a date histogram ({kibana-pull}184012[#184012]).
```

Comment:
This is quite important too and falls to the same category as the others. Let's group them together in the features section!

Reply:
I grouped them all together in my latest commit. Note that the whole filtering improvements are also covered in a better way in the release highlights https://github.com/elastic/kibana/pull/188767